test/e2e/scaffold/ssl.go

```diff
@@ -29,8 +29,8 @@ kind: Secret
 metadata:
   name: %s
 data:
-  cert: %s
-  key: %s
+  cert.pem: %s
+  key.pem: %s
```

Comment:
I modified this for webhook e2e case, not sure if it will break other e2e test cases.

Reply:
Yes ，it will break other e2e test cases. Suggest to keep `cert` and `key`, add `cert.pem` and `key.pem`.